guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -14,7 +14,7 @@ Note that legacy subscription management, including the need to attach subscript
 endif::[]
 
 ifndef::orcharhino[]
-To create, manage, and export a Red{nbsp}Hat Subscription Manifest in the Customer Portal, see https://access.redhat.com/documentation/en-us/red_hat_subscription_management/2022/html/using_red_hat_subscription_management/using_manifests_con[Using Manifests] in _Using Red Hat Subscription Management_.
+To create, manage, and export a Red{nbsp}Hat Subscription Manifest in the Customer Portal, see https://access.redhat.com/documentation/en-us/subscription_central/2021/html/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
```

Comment:
The change looks good to me, but I wonder if we should explicitly say `Satellite Server` instead of a `{ProjectServer}` since there is no upstream equivalent. @maximiliankolb What do you think?
```suggestion
To create, manage, and export a Red{nbsp}Hat Subscription Manifest in the Customer Portal, see https://access.redhat.com/documentation/en-us/subscription_central/2021/html/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected Satellite Server] in _Subscription Central_.
```

Reply:
I have no strong preference, but slightly lean towards the attribute. If it's only relevant for Satellite, maybe change line 16 to "if satellite"?

Reply:
Agreed :+1: